README.md

```diff
@@ -51,9 +57,9 @@ identity messages **MUST** conform to the following rules:
 
  - Has a `type` field mapping to a BFE string (i.e. `<06 00> + data`)
  which can assume only one the following possible values:
-   - `networkidentity/add`
-   - `networkidentity/tombstone`
- - Has a `networkidentity` field mapping to a BFE "network identity
+   - `network-identity/add`
+   - `network-identity/tombstone`
+ - Has a `network-identity` field mapping to a BFE "network identity
    ID", i.e. `<07 01> + data`
```

Comment:
`<07 01>` needs to be put in ssb-bfe-spec

Reply:
Yes, I have a branch for that, wanted to align on this spec first